firefox/MozillaFirefox.spec

```diff
@@ -113,15 +128,14 @@ BuildRequires:  makeinfo
 BuildRequires:  mozilla-nspr-devel >= 4.31
 BuildRequires:  mozilla-nss-devel >= 3.66
 BuildRequires:  nasm >= 2.14
-BuildRequires:  nodejs >= 10.22.1
+BuildRequires:  nodejs10 >= 10.22.1
```

Comment:
For TW I had to remove the explicit nodejs10 reference but it seemed to work on older distributions as well.

Reply:
I saw, but it didn't work for me, but I can't remember just now if maybe that was a SLE-12 thing.